### PR TITLE
Feature: Support for TYPO3 ContentBlocks

### PR DESCRIPTION
**Pull request by extcode**

### CODE REVIEW

File: Tests/Functional/ContentBlocksTest.php [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+class ContentBlocksTest extends FunctionalTestCase
+{
+    use TestingFramework;
+
+    protected function setUp(): void
+    {
+        $this->coreExtensionsToLoad = [
+        ];
+
```
Comment (justusmoroni) [inline Tests/Functional/ContentBlocksTest.php:40]:
```suggestion
```

**Review by justusmoroni — CHANGES_REQUESTED** [review]

File: Tests/Fixtures/content_blocks_example/ContentBlocks/ContentElements/image-fixed-width/Source/Language/Labels.xlf
Comment (justusmoroni) [inline Tests/Fixtures/content_blocks_example/ContentBlocks/ContentElements/image-fixed-width/Source/Language/Labels.xlf:1]: Replace tabs with 4 spaces.

File: Tests/Fixtures/content_blocks_example/ContentBlocks/ContentElements/image/Source/Language/Labels.xlf
Comment (justusmoroni) [inline Tests/Fixtures/content_blocks_example/ContentBlocks/ContentElements/image/Source/Language/Labels.xlf:1]: Replace tabs with 4 spaces.

**Review by justusmoroni — CHANGES_REQUESTED** [review]

File: Tests/Functional/ContentBlocksTest.php (hunk 1 above)
Comment (d-g-codappix) [reply Tests/Functional/ContentBlocksTest.php]: Should be fixed in d54310d.

File: Tests/Fixtures/content_blocks_example/ContentBlocks/ContentElements/image-fixed-width/Source/Language/Labels.xlf
Comment (d-g-codappix) [reply Tests/Fixtures/content_blocks_example/ContentBlocks/ContentElements/image-fixed-width/Source/Language/Labels.xlf:1]: Should be fixed in d54310d.

File: Tests/Fixtures/content_blocks_example/ContentBlocks/ContentElements/image/Source/Language/Labels.xlf
Comment (d-g-codappix) [reply Tests/Fixtures/content_blocks_example/ContentBlocks/ContentElements/image/Source/Language/Labels.xlf:1]: Should be fixed in d54310d.

**Review by justusmoroni — APPROVED** [review]

**Pull request merged**